extensions/crossmodel-lang/src/language-server/cross-model-serializer.ts

```diff
@@ -214,4 +226,14 @@ export class CrossModelSerializer implements Serializer<CrossModelRoot> {
       const right = obj.expression.right.$type === StringLiteral ? quote(obj.expression.right.value) : obj.expression.right.value;
       return [left, obj.expression.op, right].join(' ');
    }
+
+   private isCustomProperty(obj: any): obj is { name: string; value?: string } {
```

Comment:
@martin-fleck-at 

Can't you check the type of AST node here?

This code looks a bit tricky to me.

Reply:
I can look into it but it would mean refactoring the serialization a bit. Currently, we strip out the all properties that are not to be serialized BEFORE we go through the properties so at this point in time we no longer have that information. However, I'm sure that can be improved.

Reply:
Ok, let's leave it for now. Maybe this wil change anyway when we upgrade Langium since it now has indent support?

Reply:
As far as I can tell, we only get the support in grammar and parsing but not in serialization since there is no generic model serialization yet. But I'll know for sure once I look deeper into it.